xnmt/translator.py

```diff
@@ -236,12 +237,33 @@ def __call__(self, translator, dec_state, src, trg):
 class TranslatorReinforceLoss(Serializable):
   yaml_tag = '!TranslatorReinforceLoss'
 
+  class Regressor():
```

Comment:
Regressor can be implemented by a single Linear Transformation object located in linear.Linear.